Source/2_Core/Models/ClanPlayer.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using BeatLeader.Components;
+using BeatLeader.DataManager;
+using Newtonsoft.Json;
+
+namespace BeatLeader.Models {
+    public class ClanPlayer : IScoreRowContent {
```

Comment:
would be better to move some parts of this class into the abstraction like 'ScoreBase' or move the hidden player instance somewhere else like 'Player.CreateHidden(id)'